client/docker-compose.yml

```diff
@@ -0,0 +1,10 @@
+version: "3.8"
+services:
+  client:
+    container_name: client_container
+    build:
+      context: .
+      dockerfile: Dockerfile
+    ports:
+      - "4200:4200"
+    command: ["npm", "run", "dev"]
```

Comment:
Why are we running `npm run dev` twice for the `client/`? We're doing it for the last instruction of the Dockerfile and on `up` for `docker-compose`